ledger-api/grpc-definitions/com/daml/ledger/api/v1/admin/participant_pruning_service.proto

```diff
@@ -9,9 +9,6 @@ option java_outer_classname = "ParticipantPruningServiceOuterClass";
 option java_package = "com.daml.ledger.api.v1.admin";
 option csharp_namespace = "Com.Daml.Ledger.Api.V1.Admin";
 
-// Status: experimental interface, will change before it is deemed production
```

Comment:
I think is fine to remove this, however, I would not want to do it as part of this PR. I would like his to be discussed and removed on a separate PR.

Reply:
Reverted